src/set_theory/game/pgame.lean

```diff
@@ -148,6 +148,16 @@ def move_right : Π (g : pgame), right_moves g → pgame
 @[simp] lemma right_moves_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).right_moves = xr := rfl
 @[simp] lemma move_right_mk {xl xr xL xR} : (⟨xl, xr, xL, xR⟩ : pgame).move_right = xR := rfl
 
+/-- Conway induction on games. -/
+@[elab_as_eliminator] def move_rec_on {C : pgame → Sort} (x : pgame)
+  (IH : ∀ (y : pgame), (∀ i, C (y.move_left i)) → (∀ j, C (y.move_right j)) → C y) : C x :=
+begin
+  cases x with xl xr xL xR,
+  apply pgame.rec_on,
+  intros yl yr yL yR IHl IHr,
+  apply IH, exact IHl, exact IHr
```

Comment:
Can you combine this into a single `exact`, or does elaboration resist?

Reply:
I couldn't get it to work for whatever reason.